src/main/java/ca/warp7/frc2022/subsystems/Launcher.java

```diff
@@ -30,7 +44,8 @@ private Launcher(){
 
     @Override
     public void periodic() {
-        // currentRPS = flywheelMasterNeo.getEncoder().getVelocity() / kFlywheelGearRatio / 60;
+        //Returns in ms
+        currentRPS = launcherMotorMaster.getSelectedSensorVelocity() / 60;
```

Comment:
Make sure you get the correct calculation here! Falcons work differently than NEOs. It should probably be `launcherMotorMaster.getSelectedSensorVelocity() / kTicksPerRotation / kVelocityMeasurementPeriod / kFlywheelGear Ratio`. The falcon has 2048 ticks per rotation and measures the velocity every 0.1 seconds. You should also find and divide by the gear/belt ratio.

Reply:
Kudos for the clarification.  I tried to read the Falcon documentation last night, for my own understanding, and I thought that's how it worked....  but I wasn't sure.  This is a link to constants that Yu mentioned:

https://github.com/Team865/FRC-2022/blob/main/src/main/java/ca/warp7/frc2022/subsystems/drivetrain/FalconDriveTrainVariant.java#L12